### PR TITLE
added html and css for home page

### PR DESCRIPTION
**Pull request by rahulcs754**
* Created homepage for Coding Quiz App

image

### CODE REVIEW

File: assets/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/* color palattes used in this project */
+:root{
+    --color1: #EEEEEE;
+    --color2: #D8D2CB;
+    --color3: #398AB9;
+    --color4: #1C658C;
```
Comment (Sameony) [inline assets/css/custom.css]: describe the color classes a bit more like:
``` --gold: #EEEEEE;
    --silver: #D8D2CB;```

File: assets/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.img-c {
+    width: 100%;
+    height: 18.875rem;
+    object-fit: cover;
+}
+
+.section {
+    display: flex;
```
Comment (krishanu2911) [inline assets/css/custom.css]: just a suggestion to move it's properties to utils

**Review by krishanu2911 — COMMENTED** [review]
everything seems good to go just added a few suggestions

**Review by Sameony — APPROVED** [review]
given a minor suggestion

**Review by krishanu2911 — APPROVED** [review]

**Pull request merged**